www/docs/nextjs/setup.mdx

```diff
@@ -96,9 +87,10 @@ If strict mode is too harsh, you'll at least want to enable `strictNullChecks`:
 
 ### 3. Create a tRPC router
 
-Initialize your tRPC backend using the `initTRPC` function and create your first router. We're going to make a simple "hello world" router and procedure here - but be sure to browse the rest of the docs for information about creating more complex APIs.
+Initialize your tRPC backend in `src/server/trpc.ts` using the `initTRPC` function, and create your first router. We're going to make a simple "hello world" router and procedure here - but for deeper information on creating your tRPC API you should refer back to the [quickstart guide](/docs/quickstart) and the [Backend usage docs](/docs/server/procedures)
```

Comment:
just chose an arbitrary link for the backend usage as it was broken and we dont have an introduction/setup page for that. should we add one of those?

Reply:
I think we can link to `/server/introduction` and then it's a sensible starting point with the generated index, but also sensible if we decide to write something up properly there